lib/ansible/module_utils/manageiq.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+#
+# (c) 2017, Daniel Korn <korndaniel1@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
```

Comment:
module_utils files should be licences under BSD

https://docs.ansible.com/ansible/dev_guide/developing_modules_in_groups.html#before-you-start-coding